packages/components/src/table/index.js

```diff
@@ -323,7 +323,7 @@ class TableCard extends Component {
 								return null;
 							}
 							return (
-								<MenuItem key={ key } onInvoke={ this.onColumnToggle( key ) }>
+								<MenuItem key={ key } tabIndex={ null }>
```

Comment:
What does the `null` here do? Will this revert to the default prop? Is there a way to be more explicit (or remove altogether)?